### PR TITLE
Implement Audio Player component

### LINKED ISSUE #37: Create Audio Player Component

### PR DESCRIPTION
**Pull request by artemmatiushenko1**
Closes #37

### CODE REVIEW

File: client/src/components/audio-player/PlaybackControls.tsx [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+import CustomIconButton from '../icon-button';
+
+const { PlaybackTime } = s;
+
+const minTwoDigits = (n: number) => {
+  return (n < 10 ? '0' : '') + n;
+};
+
+const formatTime = (time: number) => {
+  const minutes = Math.floor(time / 60);
```
Comment (KhrapkoVasyl) [inline client/src/components/audio-player/PlaybackControls.tsx]: we divide any value of time by 60? magic number

File: client/src/components/audio-player/PlaybackControls.tsx [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+  const seconds = Math.floor(time - minutes * 60);
```
Comment (KhrapkoVasyl) [inline client/src/components/audio-player/PlaybackControls.tsx]: here 60 is magic number

**Review by KhrapkoVasyl — CHANGES_REQUESTED** [review]
2 small suggestions

**Review by KhrapkoVasyl — APPROVED** [review]
Not perfect but good🐷

**Review by Bogdan-Zinovij — APPROVED** [review]

**Review by DanilByaliy — APPROVED** [review]

**Pull request merged**